Sanitize the estateId received as parameter

Nowadays, when using the endpoint '/v2/estates/:id', if the `id` provided is not found in the data stored in memory, it thinks it has been dissolved and calls `getDissolvedEstate`. It checks it's already in memory, then checks if it goes beyond the last id in memory and lastly, if those checks fail, it goes to the subgraph with the query:

`{
        nfts(
          where: {
            tokenId: ${estateId}
            category: estate
            searchEstateSize: 0
          }
        ) {
          name
          estate {
            data {
              description
            }
          }
        }
      }`

Let's add a check that the `estateId` passed is a proper string to avoid requests with invalid parameters going through the graph and triggering alerts due to the 500's thrown by the server

Here's an example of a log of an invalid id used as parameter:
```
[0-fb789a0ff91ed34300cecf61d0e20236-0000000000000000-0] 2023-05-16T09:40:42.557Z [ERROR] (thegraph-port): Error:	{"queryId":"bcf55ac8-f199-4e5a-8f28-55c25e09bcb8","currentAttempt":3,"attempts":4,"timeoutWait":50000,"url":"https://subgraph.decentraland.org/marketplace","errorMessage":"GraphQL Error: Invalid response. Errors:\n- Unexpected `unsupported integer \"4802\\\"929495\"`\nExpected `IntValue`, `FloatValue`, `StringValue`, `BlockString`, `true`, `false`, `null` or `Name`. Provider: Satsuma","query":"{\n        nfts(\n          where: {\n            tokenId: 4802\"929495@\n            category: estate\n            searchEstateSize: 0\n          }\n        ) {\n          name\n          estate {\n            data {\n              description\n            }\n          }\n        }\n      }","variables":"{}"}
```

--- a/src/modules/api/component.ts
+++ b/src/modules/api/component.ts
@@ -649,6 +649,9 @@ export async function createApiComponent(components: {
   }
 
   async function getDissolvedEstate(estateId: string): Promise<NFT | null> {
+    if (estateId && !estateId.match(`^[0-9]+$`)) {
+      return null
+    }
     const { nfts } = await subgraph.query<{ nfts: DissolvedEstateFragment[] }>(
       `{
         nfts(

--- a/src/modules/map/component.ts
+++ b/src/modules/map/component.ts
@@ -244,6 +244,11 @@ export async function createMapComponent(components: {
 
   const notFoundDissolvedEstateIds = new Set<string>()
   async function getDissolvedEstate(id: string) {
+    //if id is not a tokenId
+    if (id && !id.match(`^[0-9]+$`)) {
+      return null
+    }
+
     // if this id has been queried already return before hitting the subgraph
     if (notFoundDissolvedEstateIds.has(id)) {
       return null

